pkg/storage/storage.go

```diff
@@ -0,0 +1,45 @@
+package storage
+
+import (
+	"github.com/oam-dev/kubevela/pkg/storage/driver"
+)
+
+// Storage is common storage client，use it to get app and others resource
+type Storage struct {
+	driver.Driver
+}
+
+// NewStorage form driver type
+func NewStorage(driverName string) *Storage {
+	// TODO remove driverName param ,should use environment get it
+	// FIXME use env to get user storageDriver
```

Comment:
using driverName as param is good for me.

Reply:
OK, why do you think DriverName is more friendly?

Reply:
we can make it as system config using ways more than from ENV. I don't want we rely on the only way "from ENV"

Reply:
Got it.